app/models/preserved_copy.rb

```diff
@@ -24,7 +24,9 @@ class PreservedCopy < ApplicationRecord
     REPLICATED_COPY_NOT_FOUND_STATUS => 8
   }
 
-  after_create(&:create_zipped_moab_versions!)
```

Comment:
aw, i liked this better!

Reply:
also, what if we update an existing version from 4 to 5?

Reply:
It won't matter in "part 2" where we call 2 things from this hook, so we need the block.  When we update from 4 to 5 we create a new PC row.